sdk/python/feast/infra/online_stores/helpers.py

```diff
@@ -12,13 +13,16 @@
 from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
 
 
-def get_online_store_from_config(online_store_config: Any) -> OnlineStore:
```

Comment:
I think it's better to keep this function unchanged.
Instead this function should be called with guard, eg
```
online_store = get_online_store_from_config(online_store_config) if online_store_config is not None else None
```

Reply:
I did the change, however I needed to remove the return type in `get_online_store_from_config`. There was a type mismatch in `passthrough_provider.py` initialization. This below line assumes `Optional[OnlineStore]` is the return type from `get_online_store_from_config` if I don't remove the return type. 

`self.online_store = (
            get_online_store_from_config(config.online_store)
            if not isinstance(config.online_store, Dict)
            else None
        )`

Reply:
Not sure I got it. Yes, `self.online_store` would be `Optional[OnlineStore]`. `get_online_store_from_config` signature can remain the same.

Reply:
I see your point. changed it.